sda-commons-server-openapi/src/main/java/org/sdase/commons/server/openapi/OpenApiBundle.java

```diff
@@ -36,9 +36,9 @@
  * A bundle used to generate and serve API-first or code-first OpenAPI 3 files at the {@code
  * openapi.yaml} or {@code openapi.json} HTTP resources.
  *
- * <h3>Example Usage</h3>
+ * <p>Example Usage:
```

Comment:
Shouldn’t the paragraph be closed with </p>?

Reply:
Javadoc is not HTML; a single `<p>` is correct according to [SO](https://stackoverflow.com/questions/33787753/javadoc-closing-tags) / the official documentation.